stable_baselines3/common/torch_layers.py

```diff
@@ -182,8 +184,8 @@ def __init__(
     ) -> None:
         super().__init__()
         device = get_device(device)
-        policy_net: List[nn.Module] = []
-        value_net: List[nn.Module] = []
+        policy_net: OrderedDict[str, nn.Module] = OrderedDict()
+        value_net: OrderedDict[str, nn.Module] = OrderedDict()
```

Comment:
I think we want a hook before the policy_net and the value_net as well. `pre_policy_net` or something.